src/Behat/Mink/Driver/BrowserKitDriver.php

```diff
@@ -883,7 +883,8 @@ private function mergeForms(Form $to, Form $from)
 
             if (!($field instanceof InputFormField && in_array(
```

Comment:
The multi-line function call (`in_array` in this case) within an IF isn't something you see every day. Maybe we should move it out to separate variable to reduce sequential `)` count in the IF.